2017/hipchat-breakin.md

```diff
@@ -0,0 +1,244 @@
+Meldungen, in denen [eine Verringerung der Nutzerzahlen][fb-decline]
+prognostiziert wird.
+
+Zweitens: der Anbieter und die Entwickler hinter einem Produkt! **Alle Menschen der
+Softwareindustrie tragen [eine Verantwortung][iug]**. Jeder Schnippsel Code verändert
+die Softwarelandschaft zum positiven oder negativen. Es ist im 21. Jahrhundert
+einfach nicht mehr akzeptabel, dass Programmierer irgendwelche Tools auf Github
+veröffentlichen, die dann in Monopol- oder Militärsoftware integriert werden!
+Auch hier ist noch ein riesen Paket Aufklärungsarbeit zu leisten, um
+Goldrauschprogrammierern ihrer Mitverantwortung bewusst zu machen.
+
+Die Informatik muss sich öffnen und aus der Blase hinaustreten. Es ist schön zu einem kleinen
+Kreis Eingeweihter zu gehören und mit dem eigenen Wissen viel Geld machen zu
+können – langfristig gesehen ist es aber fragwürdig, ob wir
+[Algorithmen, die von einem homogenen Bruchteil der Bevölkerung entworfen wurden][ai-bias],
+gesellschaftliche Prozesse anvertrauen wollen!
+
+## Tja, und nu?
+An dieser Stelle ist es schwierig einen Bogen auf das eigentliche Thema zurückzuschlagen.
+Ich hoffe es ist ersichtlich, dass diese Diskussion sehr schnell größere Dimensionen
+annimmt. Seit Urzeiten kann man bei jedem neuen Zwischenfall (Datenklau, weltweite
+Systemausfälle, Missbrauch…) ein weltweites [Hand-gegen-Stirn-Klatschen]
+[facepalm] hören – aber trotzdem bewegen sich die Dinge gefühlt schneller
+in Richtung unsicherer Systeme.
+
+Menschen sind genervt von Datenschutz-Diskussionen. Die Politik hält
+Datenschutz (d.h. Verbraucherschutz!) und Software-Regularien für überholt,
+die deutsche Wirtschaft sieht sich im Vergleich zum Silicon Valley im
+Nachteil – und geht lieber Risiken ein, um schneller Produkte auf den deutschen
+Markt zu bringen.
+
+Aber ist das Aufsetzen von Scheuklappen zukunftsfähig?
+Was passiert, wenn der Markt erstmal einen Punkt erreicht hat, an dem es keine
+Rückkehr mehr gibt?
+Wenn politische Systeme, die den einfachen Bürgern eigentlich ein Mitspracherecht
+garantieren sollen, von großen Konzernen gelenkt werden können?
+
+Fakt ist: jeder der heute [Daten an Dienste sendet][spiegel-echo-look] und somit dem Anbieter
+ermöglicht private Daten wirtschaftlich zu nutzen, zahlt nicht nur mit diesen, sondern legt Byte für
+Byte den Weg in die eine oder andere Richtung. **Alle, die ein Softwareprodukt
+nutzen geben implizit ihre Zustimmung** zur Art und Weise, wie dieser Dienst
+funktioniert und handelt. Und welche Folgen sich daraus für die
+Gesellschaft ergeben. Es ist keine gültige Ausrede, dass „es keine Alternative
+gäbe“ oder gar Kritik geäußert würde, indem man auf Facebook Bilder mit „ich
+widerspreche der Nutzung meiner Daten durch Facebook“ verbreitet. Im Gegenteil,
+die Plattform interessiert das Null, aber man selbst ist verlockt zu denken etwas
+getan zu haben.
+
+Daher mein Appell: Die IT muss sich ihrer wachsenden Verantwortung bewusster werden
```

Comment:
Ich habe den Eindruck, der Absatz vorher richtet sich eher an Nutzer*Innen, der Appell aber an "die IT". Vielleicht könnte man den Appell auch in zwei Teile gliedern 1. User, 2. IT.